install/kubernetes/cilium/values.yaml

```diff
@@ -191,6 +191,12 @@ updateStrategy:
 
 # Configuration Values for cilium-agent
 
+aksbyocni:
```

Comment:
After #20066 was merged, now these changes need to be made to `install/kubernetes/cilium/values.yaml.tmpl` instead, then run `make -C install/kubernetes` to update this file.

Reply:
Done 👍🏻